src/app/Plans/components/Wizard/VMConcernsIcon.tsx

```diff
@@ -2,15 +2,28 @@ import * as React from 'react';
 import { IVMwareVM } from '@app/queries/types';
 import { StatusIcon, StatusType } from '@konveyor/lib-ui';
 import { InfoCircleIcon } from '@patternfly/react-icons';
-import { getMostSevereVMConcern } from './helpers';
+import { Flex, FlexItem, Spinner } from '@patternfly/react-core';
+import './VMConcernsIcon.css';
 
+import { getMostSevereVMConcern } from './helpers';
 interface IVMConcernsIconProps {
   vm: IVMwareVM;
 }
 
 const VMConcernsIcon: React.FunctionComponent<IVMConcernsIconProps> = ({
   vm,
 }: IVMConcernsIconProps) => {
+  if (vm.revisionAnalyzed < vm.revision) {
+    return (
+      <Flex spaceItems={{ default: 'spaceItemsSm' }}>
```

Comment:
Can't we just use a `<StatusIcon status={StatusType.Loading}>` here?

Reply:
Of course we can!